### ISSUE TITLE
Request: Add storageClass as config option for workspaces

### ISSUE THREAD

**Issue by bs-matil** [eclipse-che/che#9941]
### Description
Right now one cant set the following in a PersistentVolumeClaim of a che workspace
```
annotations:
    volume.beta.kubernetes.io/storage-class: "MyStorageClass"
```

This forces the user to e.g. set a global default which has to fit che and anything else (which is unlikely) 
Or to disable the persistence

Can you please add an option to configure the storage class?

**OS and version:**    

Che 6.6.x

**Comment by diogosaad** [comment]
+1

**Comment by sleshchenko** [comment]
@jingjing188 Thanks for sharing changes. It would be kind from your side to create a PR with them.

**Comment by monaka** [comment]
Will be fixed by #12818.
